modules/porous_flow/doc/content/source/dirackernels/PorousFlowPointSourceFromPostprocessor.md

```diff
@@ -0,0 +1,18 @@
+# PorousFlowPointSourceFromPostprocessor
+
+`PorousFlowPointSourceFromPostprocessor`
+implements a mass point source that adds (removes) fluid at a
```

Comment:
i prefer "(or removes)" over the more terse "(removes)".  Probably "which was" can be replaced by "that was"